### PR TITLE
contrib/example-backend: secure cookies

### PR DESCRIPTION
**Pull request by s-urbaniak**

### CODE REVIEW

File: contrib/example-backend/options/cookie.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	SigningKey    string
+	EncryptionKey string
+}
+
+func NewCookie() *Cookie {
+	return &Cookie{}
+}
+
+func (options *Cookie) AddFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&options.SigningKey, "cookie-signing-key", options.SigningKey, "The key which is used to sign cookies, base64 encoded. Valid lengths are 32 or 64 bytes.")
```
Comment (sttts) [inline contrib/example-backend/options/cookie.go:36]: as what? base64?

Comment (sttts) [reply contrib/example-backend/options/cookie.go:36]: this should rather be a file for security, i.e. `cookie-signing-key-file`.

Comment (sttts) [reply contrib/example-backend/options/cookie.go:36]: this should rather be a file for security, i.e. `cookie-signing-key-file`.

Comment (s-urbaniak) [reply contrib/example-backend/options/cookie.go:36]: yes 👍 it's in the option help `The key which is used to sign cookies, base64 encoded.`

**Review by sttts — APPROVED** [review]

File: contrib/example-backend/options/cookie.go (hunk 1 above)
Comment (s-urbaniak) [reply contrib/example-backend/options/cookie.go:36]: > file for security

let's do this as a follow-up, there is prior art which uses CLI options. Having said that, I'd like to remove the cookie handling at all in a refactoring.

**Pull request merged**